src/main/java/com/flagsmith/interfaces/FlagsmithSdk.java

```diff
@@ -0,0 +1,46 @@
+package com.flagsmith.interfaces;
+
+import com.flagsmith.FlagsAndTraits;
+import com.flagsmith.TraitRequest;
+import com.flagsmith.config.FlagsmithConfig;
+import com.flagsmith.flagengine.environments.EnvironmentModel;
+import com.flagsmith.flagengine.features.FeatureStateModel;
+import com.flagsmith.flagengine.identities.traits.TraitModel;
+import java.util.List;
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+
+public interface FlagsmithSdk {
```

Comment:
I think this can be updated to remove the unnecessary methods as we discussed this morning?

Reply:
The unneeded methods were removed.